io.vikunja.Vikunja.yml

```diff
@@ -1,13 +1,14 @@
 app-id: io.vikunja.Vikunja
 
 base: org.electronjs.Electron2.BaseApp
-base-version: '22.08'
+base-version: '23.08'
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: '23.08'
 
 sdk: org.freedesktop.Sdk
 sdk-extensions:
   - org.freedesktop.Sdk.Extension.node18
+  #- org.freedesktop.Sdk.Extension.typescript
```

Comment:
Why the comment here?

Reply:
Nothing, just forgot to delete it. I initially tried version 24.08 and it wasn't working, so I thought it was related to this [comment](https://github.com/flathub/org.freedesktop.Sdk.Extension.node18/issues/19#issuecomment-1711844240). Turns out that there's no version 24.08 of node18 in the first place, which is why I'm using 23.08.

Reply:
Gotcha. Please remove the comment then.